classes/01class/exercises/c01-git03/zarajoy/c01-git03.md

```diff
@@ -0,0 +1,16 @@
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+Yes. The repo size was more than what I expected after I added the big file (eg increase in size was more than the size of the file i added). 
+
+2. What is the reason for this problem?
+A copy of the file was stored on the history on the folder /.git/objects with the 'git add file.txt; git commit -m "Add big file"' command
+
+3. How do you remove something from Git history after it is pushed to the remote repository? Which commands would you use? 
+git filter-branch --index-filter "git rm -rf --cached --ignore-unmatch path_to_file" HEAD
```

Comment:
What's this command doing?

Reply:
thank you, i will admit i saw the solution given online and did not break it down to understand it. Thank you for prompting me to work it out. 
Translated into plain English the command is saying (from what i understand):
In git (git), Rewrite revision history (filter-branch) and rewrite the index (--index-filter). Remove file from working directory (RM), remove file from staging area (--cached), exit gracefully if the file does not exist (--ignore-unmatch). The file (-f not -rf) we are talking about is is XXX (path_to_file). Do this for the last commit (HEAD).

Reply:
Can you add that extra information to your answer?